src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -72,13 +73,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
         .map { case (root, f) => (root, stripExtensionIfModuleFile(f)) }
         .filter { case (root, f) => fileNameFilter(f.getName) }
         .map(relativize)
+        .map(fixRelativePathForWindows)
         .map(relativeFileToLoadPath)
 
     def relativize(baseAndFile: (File, File)): String = {
       val (base, file) = baseAndFile
       base.toURI().relativize(file.toURI()).getPath()
     }
 
+    def fixRelativePathForWindows(path: String): String =
+      if (runningOnWindows) path.replace('/', File.separatorChar)
```

Comment:
Yeah, I understand you. I guess I just don't like the OS detection approach. And I completely understand that it's pretty common in the JVM world, I still don't find the OS detection based on searching a specific keyword as substring of the OS name reliable enough. And if it's possible to avoid that (even by a price of a small useless operation), I would do that.